{:type :invoke, :f :read, :value nil, :time 216528116, :process 0, :index 0}
{:type :info, :f :read, :value nil, :time 420332151, :process 0, :exception {:via [{:type org.neo4j.driver.exceptions.ServiceUnavailableException, :message "Unable to connect to localhost:7687, ensure the database is running and that there is a working network connection to it.", :at [org.neo4j.driver.internal.util.Futures blockingGet "Futures.java" 144]} {:type org.neo4j.driver.internal.shaded.io.netty.channel.AbstractChannel$AnnotatedConnectException, :message "Connection refused: localhost/127.0.0.1:7687"} {:type java.net.ConnectException, :message "Connection refused", :at [sun.nio.ch.SocketChannelImpl checkConnect "SocketChannelImpl.java" -2]}], :trace [[sun.nio.ch.SocketChannelImpl checkConnect "SocketChannelImpl.java" -2] [sun.nio.ch.SocketChannelImpl finishConnect "SocketChannelImpl.java" 777] [org.neo4j.driver.internal.shaded.io.netty.channel.socket.nio.NioSocketChannel doFinishConnect "NioSocketChannel.java" 330] [org.neo4j.driver.internal.shaded.io.netty.channel.nio.AbstractNioChannel$AbstractNioUnsafe finishConnect "AbstractNioChannel.java" 334] [org.neo4j.driver.internal.shaded.io.netty.channel.nio.NioEventLoop processSelectedKey "NioEventLoop.java" 710] [org.neo4j.driver.internal.shaded.io.netty.channel.nio.NioEventLoop processSelectedKeysOptimized "NioEventLoop.java" 658] [org.neo4j.driver.internal.shaded.io.netty.channel.nio.NioEventLoop processSelectedKeys "NioEventLoop.java" 584] [org.neo4j.driver.internal.shaded.io.netty.channel.nio.NioEventLoop run "NioEventLoop.java" 496] [org.neo4j.driver.internal.shaded.io.netty.util.concurrent.SingleThreadEventExecutor$4 run "SingleThreadEventExecutor.java" 986] [org.neo4j.driver.internal.shaded.io.netty.util.internal.ThreadExecutorMap$2 run "ThreadExecutorMap.java" 74] [org.neo4j.driver.internal.shaded.io.netty.util.concurrent.FastThreadLocalRunnable run "FastThreadLocalRunnable.java" 30] [java.lang.Thread run "Thread.java" 829]], :cause "Connection refused"}, :error "indeterminate: Connection refused: localhost/127.0.0.1:7687", :index 1}
{:type :invoke, :f :read, :value nil, :time 1012830784, :process 1, :index 2}
{:type :info, :f :read, :value nil, :time 1024944732, :process 1, :exception {:via [{:type org.neo4j.driver.exceptions.ServiceUnavailableException, :message "Unable to connect to localhost:7687, ensure the database is running and that there is a working network connection to it.", :at [org.neo4j.driver.internal.util.Futures blockingGet "Futures.java" 144]} {:type org.neo4j.driver.internal.shaded.io.netty.channel.AbstractChannel$AnnotatedConnectException, :message "Connection refused: localhost/127.0.0.1:7687"} {:type java.net.ConnectException, :message "Connection refused", :at [sun.nio.ch.SocketChannelImpl checkConnect "SocketChannelImpl.java" -2]}], :trace [[sun.nio.ch.SocketChannelImpl checkConnect "SocketChannelImpl.java" -2] [sun.nio.ch.SocketChannelImpl finishConnect "SocketChannelImpl.java" 777] [org.neo4j.driver.internal.shaded.io.netty.channel.socket.nio.NioSocketChannel doFinishConnect "NioSocketChannel.java" 330] [org.neo4j.driver.internal.shaded.io.netty.channel.nio.AbstractNioChannel$AbstractNioUnsafe finishConnect "AbstractNioChannel.java" 334] [org.neo4j.driver.internal.shaded.io.netty.channel.nio.NioEventLoop processSelectedKey "NioEventLoop.java" 710] [org.neo4j.driver.internal.shaded.io.netty.channel.nio.NioEventLoop processSelectedKeysOptimized "NioEventLoop.java" 658] [org.neo4j.driver.internal.shaded.io.netty.channel.nio.NioEventLoop processSelectedKeys "NioEventLoop.java" 584] [org.neo4j.driver.internal.shaded.io.netty.channel.nio.NioEventLoop run "NioEventLoop.java" 496] [org.neo4j.driver.internal.shaded.io.netty.util.concurrent.SingleThreadEventExecutor$4 run "SingleThreadEventExecutor.java" 986] [org.neo4j.driver.internal.shaded.io.netty.util.internal.ThreadExecutorMap$2 run "ThreadExecutorMap.java" 74] [org.neo4j.driver.internal.shaded.io.netty.util.concurrent.FastThreadLocalRunnable run "FastThreadLocalRunnable.java" 30] [java.lang.Thread run "Thread.java" 829]], :cause "Connection refused"}, :error "indeterminate: Connection refused: localhost/127.0.0.1:7687", :index 3}
{:type :invoke, :f :read, :value nil, :time 2830964037, :process 2, :index 4}
{:type :ok, :f :read, :value nil, :time 3105565284, :process 2, :index 5}
{:type :invoke, :f :read, :value nil, :time 3111862652, :process 2, :index 6}
{:type :ok, :f :read, :value nil, :time 3125036160, :process 2, :index 7}
